src/pyedb/dotnet/edb_core/utilities/simulation_setup.py

```diff
@@ -100,7 +100,11 @@ def __init__(self, pedb, edb_object=None):
 
     @property
     def sim_setup_info(self):
-        return SimSetupInfo(self._pedb, sim_setup=self, edb_object=self._edb_object.GetSimSetupInfo())
+        if self.type not in ["hfss_pi", "raptor_x"]:
+            return SimSetupInfo(self._pedb, sim_setup=self, edb_object=self._edb_object.GetSimSetupInfo())
+        else:
+            if self._edb_setup_info:
```

Comment:
@svandenb-dev why do you want to cache self._edb_setup_info for SIwave and HFSS?

Reply:
I need to cache because GetSimulationSetupInfo is missing in new classes